src/signals/incident/containers/KtoContainer/index.js

```diff
@@ -92,8 +92,7 @@ export const KtoContainer = () => {
   const { satisfactionIndication, uuid } = useParams()
   const isSatisfied = satisfactionIndication === 'ja'
   const [contactAllowed, setContactAllowed] = useState(
-    configuration.featureFlags.reporterMailHandledNegativeContactEnabled &&
-      satisfactionIndication === 'nee'
+    satisfactionIndication === 'nee'
```

Comment:
This default is needed to pass to ktoform.

Reply:
Not entirely sure why this is, thought I tested this succesfully without `configuration.featureFlags.reporterMailHandledNegativeContactEnabled` as well, but added this back. IMO no harm leaving this in.